SConstruct

```diff
@@ -421,8 +421,8 @@ def get_libs(lib, static):
             else:
                 static_libs.append(l)
         return (static_libs, dynamic_libs)
-    except:
-        raise Exception('pkg-config failed for ' + lib)
+    except Exception as e:
+        raise Exception('pkg-config failed for ' + lib + ' Exception: ' + str(e))
```

Comment:
Nit: Let's put a semicolon between the messages. So replace `' Exception: '` with `'; Exception: '`.